build/images/Dockerfile.build.coverage

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.13 as antrea-build
+
+WORKDIR /antrea
+
+COPY go.mod /antrea/go.mod
+
+RUN go mod download
+
+COPY . /antrea
+
+RUN make antrea-agent antrea-controller antrea-cni antctl-ubuntu antrea-controller-instr-binary antrea-agent-instr-binary
+
+
+FROM antrea/base-ubuntu:2.14.0
+
+LABEL maintainer="Antrea <projectantrea-dev@googlegroups.com>"
+LABEL description="The Docker image to deploy the Antrea CNI. "
```

Comment:
Use a different description, e.g.: "The Docker image to deploy the Antrea CNI with code coverage measurement enabled (used for testing)".

Reply:
Done